MOBILE_API.xml

```diff
@@ -2376,25 +2376,34 @@
         <description>Configuration of on-screen keyboard (if available).</description>         
         <param name="language" type="Language" mandatory="false">
             <description>The keyboard language.</description>
-        </param>         
+        </param>
         <param name="keyboardLayout" type="KeyboardLayout" mandatory="false" >
             <description>Desired keyboard layout.</description>
-        </param>         
+        </param>
         <param name="keypressMode" type="KeypressMode" mandatory="false" >
             <description>
                 Desired keypress mode.
                 If omitted, this value will be set to RESEND_CURRENT_ENTRY.
             </description>
-        </param>         
+        </param>
         <param name="limitedCharacterList" type="String" maxlength="1" minsize="1" maxsize="100" array="true" mandatory="false">
             <description>Array of keyboard characters to enable.</description>
             <description>All omitted characters will be greyed out (disabled) on the keyboard.</description>
             <description>If omitted, the entire keyboard will be enabled.</description>
-        </param>         
-        <param name="autoCompleteText" type="String" maxlength="1000" mandatory="false">
-            <description>Allows an app to prepopulate the text field with a suggested or completed entry as the user types</description>
-        </param>         
-    </struct>         
+        </param>
+        <param name="autoCompleteText" type="String" maxlength="1000" mandatory="false" deprecated="true" since="6.0">
+            <description>Deprecated, use autoCompleteList instead.</description>
+            <history>
+                <param name="autoCompleteText" type="String" maxlength="1000" mandatory="false" since="3.0" until="6.0" />
+            </history>
+        </param>
+        <param name="autoCompleteList" type="String" maxlength="1000" minsize="0" maxsize="100" array="true" mandatory="false" since="6.0">
+            <description>
+                Allows an app to prepopulate the text field with a list of suggested or completed entries as the user types. 
+                If empty, the auto-complete list will be removed from the screen.
```

Comment:
The [proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0108-AutoCompleteList.md) includes the following in the param description:
`Set to 0 to remove the auto-complete list from the screen`

Reply:
This description doesn't make any sense given the type of this value, so I reworded it myself. I believe that the new description is clearer